internal/daysteps/daysteps.go

```diff
@@ -12,9 +25,54 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	parts := strings.Split(data, ",")
+	if len(parts) != 2 {
+		err := fmt.Errorf("there was a problem when calling \"parsePackage()\" with argument \"%s\": %w\n", data, ErrInvalidInput)
+		return 0, 0, err
+	}
+
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		err = fmt.Errorf("there was a problem when calling \"parsePackage()\" with this argument's part \"%s\": %w\n", parts[0], ErrConvToInt)
+		return 0, 0, err
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```